theano/gpuarray/ctc.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import absolute_import, print_function, division
+
+import theano
+from theano import (config, gof)
+import theano.tensor as T
+from .basic_ops import (gpu_contiguous, as_gpuarray_variable, infer_context_name)
+import theano.tensor.nnet.ctc
+from .type import (GpuArrayType, gpu_context_type)
+from .elemwise import GpuDimShuffle
+from theano.gradient import grad_undefined
+from theano.gof import local_optimizer
+from theano.tensor.opt import register_canonicalize
+from theano.tensor.opt import register_stabilize
+
+import os
+import pygpu
```

Comment:
Hello, @joaovictortr , could you replace this line with:
```python
from . import pygpu
```
This will import pygpu from gpuarray init file, which should help prevent some errors with jenkins when building Theano documentation.